app/client/ui/PageWidgetPicker.ts

```diff
@@ -99,21 +99,33 @@ const testId = makeTestId('test-wselect-');
 
 // The picker disables some choices that do not make much sense. This function return the list of
 // compatible types given the tableId and whether user is creating a new page or not.
-function getCompatibleTypes(tableId: TableRef, isNewPage: boolean|undefined): IWidgetType[] {
+function getCompatibleTypes(tableId: TableRef,
+                            {isNewPage, summarize}: {isNewPage: boolean|undefined, summarize: boolean}): IWidgetType[] {
```

Comment:
Might be worth breaking out an interface for this params object. I see it's also duplicated below.